tables/DataMan/test/tDataManInfo.cc

```diff
@@ -211,6 +211,28 @@ void largeTest()
     AlwaysAssertExit (dminfoRes.subRecord(1).subRecord("SPEC").nfields() == 1);
     AlwaysAssertExit (dminfoRes.subRecord(1).subRecord("SPEC").asInt("BSZ") == 10);
   }
+  Record dminfoFinal = DataManInfo::finalizeMerge (desc, dminfoRes);
+  cout << dminfoFinal;
+}
+
+void testUniqueName()
```

Comment:
Please add a "FOO", "FOO", "FOO", "FOO" test, and check that the result is "FOO", "FOO_0", "FOO_1", "FOO_2".

Reply:
Done